src/components/AppMapDetailsObj.vue

```diff
@@ -19,6 +19,10 @@
         <b-btn v-show="areaMarkers.length" size="sm" block variant="dark" @click="keepAreaMarkersAlive()">Keep area representation loaded</b-btn>
         <b-btn v-show="staticData.persistentAreaMarkers.length" size="sm" block variant="dark" @click="forgetPersistentAreaMarkers()">Hide area representation</b-btn>
       </section>
+      <section class="mt-2" v-show="minObj.korok_type">
```

Comment:
Consider showing the Korok markers automatically, similar to how Area representations are shown and hidden automatically